src/main/js/dashboard-frontend/src/components/details/DetailBody.tsx

```diff
@@ -85,11 +85,11 @@ export default class DetailBody extends Component<
         <ColumnLayout columns={2} variant="text-grid">
           <SpaceBetween size={"l"}>
             <div>
-              <Box margin={{bottom: "xxxs"}} color="text-label">Component name</Box>
+              <Box margin={{bottom: "xxxs"}} variant="awsui-key-label"  color="text-label">Component name</Box>
```

Comment:
why is this needed?

Reply:
This has been suggested by YingYing, in order to have the label in bold.

Reply:
Commented on the wrong thing?

Reply:
no, no this variant is the recommendation from cloudscape ambassador.
See here: https://cloudscape.design/components/key-value-pairs/